application/core/QuestionTypes/ArrayFlexibleRow/RenderArrayFlexibleRow.php

```diff
@@ -290,20 +287,19 @@ public function getNonDropdownRows()
                     'ld' => $oAnswer->code,
                     'code' => $oAnswer->code,
                     'label' => $oAnswer->answerl10ns[$this->sLanguage]->answer,
-                    'checked' => ($this->getFromSurveySession($myfname) == $oAnswer->code) ? 'checked' : '',
```

Comment:
Is this correct? Seems unrelated to PR title?

Reply:
https://github.com/LimeSurvey/LimeSurvey/pull/3189

Reply:
I updated PR link in the Jira ticket.